collections/individual/index.php

```diff
@@ -402,32 +405,32 @@ function initializeMap(){
 						}
 						if(array_key_exists('relation',$occArr)){
 							?>
-							<fieldset id="association-div">
-								<legend><?php echo (isset($LANG['RELATED_OCCUR'])?$LANG['RELATED_OCCUR']:'Related Occurrences'); ?></legend>
-								<?php
-								$displayLimit = 5;
-								$cnt = 0;
-								foreach($occArr['relation'] as $id => $assocArr){
-									if($cnt == $displayLimit){
-										echo '<div class="relation-hidden"><a href="#" onclick="$(\'.relation-hidden\').toggle();return false;">show all records</a></div>';
-										echo '<div class="relation-hidden" style="display:none">';
+								<fieldset id="association-div" class="top-light-margin">
```

Comment:
I fixed some indentation here and added `class="top-light-margin"`. Otherwise, there are no changes in this code chunk.